file.rb

```diff
@@ -32,17 +32,18 @@ def correct_name
 end
 
 class Person < Nameble
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
```

Comment:
- [ ] I would like to recommend you make sure that all your instance variables have `attr_accessor` since there are both getter and setter.  attr_reader defines only getter.  Please fix it to make your code work as expected :+1: